src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -249,6 +252,36 @@ subroutine get_fv3_regional_ensperts_run(this,en_perts,nelen,ps_bar)
            fv3_filename%tracers=trim(ensfilenam_str)//"-fv3_tracer"
            fv3_filename%sfcdata=trim(ensfilenam_str)//"-fv3_sfcdata"
            fv3_filename%couplerres=trim(ensfilenam_str)//"-coupler.res"
+
+           if(mype == 0) then
+              call nc_check(nf90_open(fv3_filename%dynvars,nf90_nowrite,loc_id), &
+              "nf90 open ",trim(fv3_filename%dynvars))
+              call nc_check(nf90_inquire(loc_id,formatNum=ncfmt), &
+              "nf90_inquire formate of ",trim(fv3_filename%dynvars))
+              if(ncfmt .ne.nf90_format_netcdf4) then
```

Comment:
@MichaelLueken-NOAA I think the failure for the regression test netcdf3_fv3_regional is just because in the current regression test, the fv3-lam files are in netcdf3, which was forbid  by the test added in this PR.  I would suggest to change the background files in this regression to netcdf4 format.  Please let me know if that works from your point of view.

Reply:
@TingLei-daprediction I was able to use `nccopy -k nc4` on the NetCDF files for the netcdf_fv3_regional canned case, updated the netcdf_fv3_regional.sh script to use the nc4 files rather than nc files, then resubmitted the test.  As expected this test successfully ran.  I'm fine with converting the NetCDF files to NetCDF-4 for the netdf_fv3_regional test.  Please go into this script and change the *.nc files to converted *.nc4 files.

Reply:
@TingLei-daprediction I go into build/regression/CTestTestfile.cmake and comment out the lines that aren't for the specific test I want to run.  You should also be able to use:

`ctest -I 16,16`

to submit just the netcdf_fv3_regional test.

Reply:
@MichaelLueken-NOAA  maybe the question should be asked on the GSI github. Now, how can the regression test function be enabled ?  I tried to change  to 
`option(BUILD_REG_TESTING "Build the Regression Testing Suite" ON)` 
But the cmake command would give message 
`CMake Warning at regression/CMakeLists.txt:14 (message):
  RT: Control GSIexec Not Found, GSI Regression Tests Disabled!` 
Your further help is appreciated.

Reply:
@TingLei-daprediction I don't think I understand what you are asking here.  The regression tests can't be run on GitHub, only on Hera and WCOSS1 (and only Hera, after today).

Reply:
@TingLei-daprediction Not a problem.  The entry in CMakeLists.txt should remain as:

`option(BUILD_REG_TESTING "Build the Regression Testing Suite" OFF)`

In `ush/build.sh`, the following lines dictate the activation of the regression tests on Hera:

`REGRESSION_TESTS=${REGRESSION_TESTS:-"YES"} # Build regression test suite`

```
elif [[ $MACHINE_ID = hera.intel ]] ; then
    CONTROLPATH="/scratch1/NCEPDEV/da/Michael.Lueken/svn1/install/bin"
```

```
# Build regression test suite (on supported MACHINE_ID where CONTROLPATH exists)
[[ ${REGRESSION_TESTS} =~ [yYtT] ]] && CMAKE_OPTS+=" -DBUILD_REG_TESTING=ON -DCONTROLPATH=${CONTROLPATH:-}"
```

An `ls` of the `svn1/install/bin` on Hera gives:

```
[Michael.Lueken@hfe08 ush]$ ls /scratch1/NCEPDEV/da/Michael.Lueken/svn1/install/bin
enkf.x  gsi.x
```

So, building the GSI and EnKF using `ush/build.sh` should set up the regression tests on Hera.  If you could point me to your local clone, I will take a look and see why you aren't able to find my `svn1/install/bin` directory to toggle on regression tests.

Reply:
@MichaelLueken-NOAA   The problem was that I still did "1 . cmake .. and 2. make .." manually. 
Using build.sh should work fine .       Thank you so much!

Reply:
@MichaelLueken-NOAA  adding "nccopy ... " in the netcdf3_fv3_regional.sh works fine and the netcdf3_fv3_regional regress test passed.  I had pushed this  and other changes you requested to the PR branch. 
Thanks.